galette/lib/Galette/Core/Galette.php

```diff
@@ -224,15 +223,19 @@ public static function getMenus(bool $public = false): array
                             'title' => _T("View and filter contributions"),
                             'route' => [
                                 'name' => 'contributions',
-                                'args' => ['type' => 'contributions']
+                                'args' => ['type' => 'contributions'],
+                                'aliases' => ['editContribution'],
+                                'sub_select' => false
```

Comment:
Here it can be a Contribution or a Donation, I cannot choose beetween contributions list any of add links, no one seems relevant. So, no sub menu will be selected.

Reply:
Here too, the list is relevant enough in my opinion, for the same reason as members.